Add rOpenScPCA-related FAQs (#991)
AlexsLemonade/OpenScPCA-analysis: docs/troubleshooting-faq/faq.md

In such cases, you will need to run the module yourself to generate the results.
Instructions for [running the module](../contributing-to-analyses/analysis-modules/running-a-module.md), including its software and compute requirements, should be available in the module's main `README.md` file.
After running the module, results will generally be stored in `analysis/{module name}/results`, and the module's documentation should describe the contents of result files.

### What if I want to use Seurat?

> While [data downloads](../getting-started/accessing-resources/getting-access-to-data.md) are only available in `SingleCellExperiment` and `AnnData` format, `Seurat` versions of all objects (in [v5 assay format](https://satijalab.org/seurat/articles/seurat5_essential_commands)) are also available for use.

Reviewer: Not all objects are available at this time.
```suggestion
While [data downloads](../getting-started/accessing-resources/getting-access-to-data.md) are only available in `SingleCellExperiment` and `AnnData` format, `Seurat` versions of processed objects (in [v5 assay format](https://satijalab.org/seurat/articles/seurat5_essential_commands)) are also available for use.
```

These files are part of the OpenScPCA results, associated with the module [`seurat-conversion`](https://github.com/AlexsLemonade/OpenScPCA-analysis/tree/main/analyses/seurat-conversion) which we wrote to convert the processed `SingleCellExperiment` objects to `Seurat` format.
For more information on obtaining result files, please refer to the documentation for [the `download-results.py` script](../getting-started/accessing-resources/getting-access-to-data.md#accessing-scpca-module-results).

When working with these `Seurat` objects, please bear in mind the following:

> * They were _not_ processed with a `Seurat` pipeline.
> They were processed using the same pipeline as all OpenScPCA objects were (e.g., with `Bioconductor`), and then converted to `Seurat` format
>   * Notably, they do contain the raw data counts, allowing you to perform normalization, dimension reduction, etc. with `Seurat` directly if you so choose

Reviewer: Rather than focusing on what is not there, I would describe what is present:
```suggestion
* These `Seurat` objects include the same content as the `SingleCellExperiment` objects that they are derived from.
This includes raw and normalized counts, annotations of highly variable genes, PCA and UMAP transformations, as well as cell and feature metadata.  
  * Note that all calculations were performed using `Bioconductor` packages, so values will differ from the results obtained using `Seurat` functions from the same raw data.
  * If `Seurat`-derived values are required, processing steps may need to be repeated.
```

(I'm not sure about the last sentence)

I'd also move the second main bullet point about feature names to be first.

> * To be more consistent with `Seurat` analysis pipelines, gene names in these objects use gene symbols rather than Ensembl ids

Reviewer: ```suggestion
* To be more consistent with `Seurat` analysis pipelines, these objects use gene symbols rather than Ensembl ids as the row names and primary feature id.
```

### The ScPCA data objects contain Ensembl ids, but I need gene symbols for my analysis. How should I perform this conversion?

In an effort to keep this consistent across the OpenScPCA project, we provide functions to convert Ensembl ids to gene symbols in an R package we maintain called [`rOpenScPCA`](https://github.com/AlexsLemonade/rOpenScPCA/).

> Installation instructions are provided in the `rOpenScPCA` GitHub repository.

Reviewer: I would either move the repo link here or just add another link that goes direct to the installation instructions.
```suggestion
Installation instructions are provided in the [`rOpenScPCA` GitHub repository](https://github.com/AlexsLemonade/rOpenScPCA/?tab=readme-ov-file#installation).
```

This package has two particular functions to support this task:

> * `rOpenScPCA::sce_to_symbols()`
>   * This function converts row names in a `SingleCellExperiment` object from Ensembl ids to gene symbols
> * `rOpenScPCA::ensembl_to_symbol()`
>   * This function converts a vector of Ensembl ids to a vector of gene symbols

Reviewer: I would reverse the order here
```suggestion
* `rOpenScPCA::ensembl_to_symbol()`
  * This function converts a vector of Ensembl ids to a vector of gene symbols
* `rOpenScPCA::sce_to_symbols()`
  * This function converts row names in a `SingleCellExperiment` object from Ensembl ids to gene symbols
```

> Please refer to these functions' help menus (e.g., `?rOpenScPCA::sce_to_symbols`) for additional information on their use.

Reviewer: ```suggestion
Please refer to these functions' help pages (e.g., `?rOpenScPCA::sce_to_symbols`) for additional information on their use, including options for handling duplicate gene symbols.
```

> Please refer to these functions' help pages (e.g., `?rOpenScPCA::sce_to_symbols`) for additional information on their use, including options for handling duplicate and/or missing gene symbols.

Reviewer: I would do one of: choose and or or, because I don't think it matters. Or just leave off the missing symbols because there are other options too, but I just wanted to highlight the one that most people will think of.

### I noticed there are cluster assignments in the processed data files. Should I use those or re-cluster the data?

All ScPCA data objects contain cluster assignments which were [calculated using an automated pipeline](https://scpca.readthedocs.io/en/stable/processing_information.html#processed-gene-expression-data).
Because the clustering parameters used in this automated pipeline were not tailored to any given dataset, we do not recommend relying on these clusters for downstream analysis.
Instead, we strongly recommend re-clustering the data _and_ evaluating your cluster assignments before using them.

To support clustering analysis and evaluation, we provide several functions in an R package we maintain called [`rOpenScPCA`](https://github.com/AlexsLemonade/rOpenScPCA/) to accomplish the following tasks:

* Perform graph-based clustering
* Evaluate clustering results with several quality control metrics
* Calculate different sets of clustering results across parameter space in order to identify an optimal clustering scheme

> We also provide an OpenScPCA analysis module [`hello-clusters`](https://github.com/AlexsLemonade/OpenScPCA-analysis/tree/main/analyses/hello-clusters) with example notebooks demonstrating how to use clustering functionality in `rOpenScPCA`.

Reviewer: ```suggestion
We also provide an OpenScPCA analysis module [`hello-clusters`](https://github.com/AlexsLemonade/OpenScPCA-analysis/tree/main/analyses/hello-clusters) with example notebooks demonstrating how to use the clustering functionality in `rOpenScPCA`.
```
